petition/models.py

```diff
@@ -0,0 +1,152 @@
+from django.db import models
+from django.utils import timezone
+
+# Creates a tag, there can be at most three in a single petition
+class Tag(models.Model):
+    label = models.CharField(max_length=15, primary_key=True)	
+
+    # Returns label when asked for the tag 
+    def __unicode__(self):
+        return self.label	
+
+# Creates a user based on their name, RCS ID and admin status
+class User(models.Model):
+    rcs_id = models.CharField(max_length=10, primary_key = True)	# RCS ID	
+    name = models.CharField(max_length=50)	# The author name
+    admin = models.BooleanField(default=False)	# Is admin?
+    banned = models.BooleanField(default=False)	# Is the person banned?-- not very likely
+    initials = models.CharField(max_length=2)		# Their initials
+    union_member = models.BooleanField(default=False)	# Is member of the union?
+    
+    # Returns their name
+    def __unicode__(self):
+        return self.rcs_id 	
+    
+    # Returns the initials of the User using the RCS ID, first strips
+    # it of the digits at the end and then 
+    def set_initials(self):
+        full_name = self.rcs_id
+        name_modified = ''.join([i for i in full_name if not i.isdigit()])
```

Comment:
Might want to .upper() name_modified so that you save the initials as capitals since the RCS ID should be all lowercase when its enter